README.md

```diff
@@ -101,6 +101,64 @@ You can enable it using:
 global.runAsNonRoot: true
 ```
 
+## Backups
+
+This chart includes optional backup of CA certs using [Restic](https://restic.net/), the default configuration supports S3 or S3 compatible storage by setting a configuration similar to this:
+
+```
+backup:
+  enabled: true
+  restic:
+    repository: "s3:https://s3.minio.xx/backups/"
+    access_key_id: "ACCESSKEYHERE"  # s3 access key
+    secret_access_key: "SECRETACCESSHERE"  # s3 secret access key
+    password: "ENCRYPTIONPASSWORDHERE"  # restic encryption password
+```
+
+Alternatively you can define `puppetserver.masters.backup.restic.repository` and `puppetserver.masters.backup.restic.existingSecret` to use a pre-configured (NOTE: this chart will not provision the secret if defined) e.g.:
+
+
+```
+backup:
+  enabled: true
+  restic:
+    repository: "s3:https://s3.minio.xx/backups/"
+    existingSecret: restic-env
+```
+
+a compatible secret can be created e.g.
```

Comment:
IMHO it would be enough to mention that the secret can contain any KEY=VALUE pairs that will be set as the environment variables for Restic, we don't need to explain how to create secrets in Kubernetes, as using an existing secret implies that you already know how to get one there manually or via your secret engine(s). Link to the Restic documentation for backend config options instead?

Reply:
I'm inclined to agree here, let me reword it and update tomorrow as well as any conclusion on the remaining readme point about puppet version

Reply:
latest commit now deals with this